resources/assets/js/components/front/char/CharListDetail.vue

```diff
@@ -0,0 +1,203 @@
+                                    <td>Stem color (before transplanting)</td>
+                                    <td>Green</td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                    <br><br>
+                    <div class="col-md-10 table-responsive passport-detail">
+                    <h4>VEGETATIVE DATA</h4>
+                        <table class="table">
+                            <tbody>
+                                <tr>
+                                    <td>Hypocotyl color (terminal bud 1-2 mm)</td>
+                                    <td></td>
+                                </tr>
+                                <tr>
+                                    <td>Hypocotyl color intensity</td>
+                                    <td></td>
+                                </tr>
+                                <tr>
+                                    <td>Cotyledonous leaf length (mm) (N=10)</td>
+                                    <td></td>
+                                </tr>
+                                <tr>
+                                    <td>Cotyledonous leaf width (mm) (N=10)</td>
+                                    <td></td>
+                                </tr>
+                                <tr>
+                                    <td>Cotyledonous leaf shape</td>
+                                    <td></td>
+                                </tr>
+                                <tr>
+                                    <td>Cotyledonous leaf color</td>
+                                    <td></td>
+                                </tr>
+                                <tr>
+                                    <td>Hypocotyl pubescence</td>
+                                    <td></td>
+                                </tr>
+                                <tr>
+                                    <td>Stem color (before transplanting)</td>
+                                    <td>Green</td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                    <br><br>
+                    <div class="col-md-10 table-responsive passport-detail">
+                        <h4>INFLORESCENCE DATA</h4>
+                    </div>
+                    <br><br>
+                    <div class="col-md-10 table-responsive passport-detail">
+                        <h4>FRUIT DATA</h4>
+                    </div>
+                    <br><br>
+                    <div class="col-md-10 table-responsive passport-detail">
+                        <h4>SEED DATA</h4>
+                    </div>
+                </div>
+            </div>
+        </div>
+      </div><!--End of container-->
+    </section>
+    </front-base>
+   
+</template>
+
+<script>
+import FrontBase from '../FrontBase';
+
+function Seedling() {
```

Comment:
You can use collapse Bootstrap components https://getbootstrap.com/docs/3.3/javascript/#collapse